shipyard-controller/docs/swagger.json

```diff
@@ -1749,6 +1763,10 @@
                     "description": "git user",
                     "type": "string"
                 },
+                "insecureSkipTLS": {
+                    "description": "insecure skip tls\nomitempty property is missing due to fallback of this\nparameter to \"undefined\" when marshalling/unmarshalling data\nwhen \"false\" value is present",
```

Comment:
Could you please adapt also in this file the new message for `insecureSkipTLS`?

Reply:
adapted, thanks!